### ISSUE TITLE
Standardize READMEs; Logos for FUNDING section

### ISSUE THREAD

**Issue by mankoff** [GEUS-Glaciology-and-Climate/.github#3]
I'd like to add a  FUNDING section to various READMEs (freshwater; solid ice; mass_balance). It seems like we might want
 
+ Some suggested standard sections for READMEs so that there is a theme and continuity across the GEUS GitHub presence
+ An online repository of graphics and logos so that we can all use a shared homogeneous graphic.

**Comment by mankoff** [comment]
Added to freshwater README https://github.com/GEUS-Glaciology-and-Climate/freshwater/blob/main/README.org#funding

**Comment by PennyHow** [comment]
> + An online repository of graphics and logos so that we can all use a shared homogeneous graphic.

Good idea to all of this. With regard to the logos and graphics repo, all are welcome to use our [github.io](https://github.com/GEUS-Glaciology-and-Climate/geus-glaciology-and-climate.github.io) repo which we have been using for figures and logos across multiple repos. Just put any graphics and logos in the assets folder.
